test/630-bridge-vlan.bats

```diff
@@ -140,3 +140,37 @@ echo "$config"
     run_netavark teardown $(get_container_netns_path) <<<"$config1"
     run_netavark teardown $(get_container_netns_path 1) <<<"$config2"
 }
+
+@test "bridge - mtu from default route" {
+    run_in_host_netns ip link add default_route type dummy
+    run_in_host_netns ip link set default_route mtu 9000
+    run_in_host_netns ip addr add 192.168.0.0/24 dev default_route
+    run_in_host_netns ip link set default_route up
+    run_in_host_netns ip route add default via 192.168.0.0
+
+    local vlan=20
+    local config=$(createVlanConfig managed $vlan)
+
+    run_netavark setup $(get_container_netns_path) <<<"$config"
+
+    # check that interface exists
+    run_in_container_netns ip -j --details link show eth0
+    link_info="$output"
+    assert_json "$link_info" ".[].mtu" "=="  "9000" "MTU matches default route MTU"
+}
+
+@test "bridge - mtu from existing bridge" {
+    run_in_host_netns ip link add podman0 type bridge
+    run_in_host_netns ip link set podman0 mtu 9000
```

Comment:
I recommend to use a different mtu value here just to have it different from the other test. Consider the case where the code hard codes to 9000 in which case both would pass but not do what we want.